metricbeat/module/apache/status/data.go

```diff
@@ -211,40 +211,50 @@ func eventMapping(body io.ReadCloser, hostname string, metricset string) common.
 	}
 
 	event := common.MapStr{
-		"totalAccesses":           totalAccesses,
-		"totalKBytes":             totalKBytes,
-		"uptime":                  uptime,
-		"cpuLoad":                 cpuLoad,
-		"cpuUser":                 cpuUser,
-		"cpuSystem":               cpuSystem,
-		"cpuChildrenUser":         cpuChildrenUser,
-		"cpuChildrenSystem":       cpuChildrenSystem,
-		"reqPerSec":               reqPerSec,
-		"bytesPerSec":             bytesPerSec,
-		"bytesPerReq":             bytesPerReq,
-		"busyWorkers":             busyWorkers,
-		"idleWorkers":             idleWorkers,
-		"connsTotal":              connsTotal,
-		"connsAsyncWriting":       connsAsyncWriting,
-		"connsAsyncKeepAlive":     connsAsyncKeepAlive,
-		"connsAsyncClosing":       connsAsyncClosing,
-		"serverUptimeSeconds":     serverUptimeSeconds,
-		"load1":                   load1,
-		"load5":                   load5,
-		"load15":                  load15,
-		"scbStartingUp":           totalS,
-		"scbReadingRequest":       totalR,
-		"scbSendingReply":         totalW,
-		"scbKeepalive":            totalK,
-		"scbDnsLookup":            totalD,
-		"scbClosingConnection":    totalC,
-		"scbLogging":              totalL,
-		"scbGracefullyFinishing":  totalG,
-		"scbIdleCleanup":          totalI,
-		"scbOpenSlot":             totalDot,
-		"scbWaitingForConnection": totalUnderscore,
-		"scbTotal":                totalAll,
-		"hostname":                hostname,
+		"hostname":      hostname,
+		"totalAccesses": totalAccesses,
+		"totalKBytes":   totalKBytes,
+		"reqPerSec":     reqPerSec,
+		"bytesPerSec":   bytesPerSec,
+		"bytesPerReq":   bytesPerReq,
+		"busyWorkers":   busyWorkers,
```

Comment:
I was thinking about them for longer and decided to leave them in main object. They are fields which are visible when you configure basic server-status. They could be grouped to something called 'status', but I don't think we need to group them because they should be indexed all the time.